static/css/custom.css

```diff
@@ -82,6 +82,24 @@
   color: white !important;
   background-color: var(--accent-color-dark);
 }
+/* Button-like links used for calls to action centered on page
```

Comment:
Cool. Definitely works, but is there a reason we'd need to have one thing that's an action-button and a second thing that's an action-button-center? Aren't they always centered? If so, this is quite a bit of duplicate code.

Reply:
For me, the centered buttons look really weird on the "Join Us". 

Here is the current layout: 

<img width="357" alt="Screen Shot 2020-09-10 at 3 06 50 PM" src="https://user-images.githubusercontent.com/62075223/92814010-47d4af00-f377-11ea-9072-176f3065371f.png">

Here is the page with everything centered:

<img width="351" alt="Screen Shot 2020-09-10 at 3 11 20 PM" src="https://user-images.githubusercontent.com/62075223/92814417-ea8d2d80-f377-11ea-80ce-fee18a435bcd.png">

If you think the latter looks ok though then I am all for simplifying.

Reply:
I see what you're saying. though the left button and small left image look a little weird to me too. too much white space to the right there, followed by a narrow band of text. maybe we need an alternate solution there. I can play with it a bit. Is there anywhere else (so far) where a centered button doesn't work?